.dev/tests/phpunit/test-class-coblocks.php

```diff
@@ -190,8 +190,6 @@ public function test_final_build_assets_exist() {
 				'dist/js/coblocks-google-recaptcha.js',
 				'dist/js/coblocks-lightbox.js',
 				'dist/js/coblocks-masonry.js',
-				'dist/js/coblocks-slick-initializer-front.js',
-				'dist/js/coblocks-slick-initializer.js',
```

Comment:
I suspect this will break existing blocks that are using the slick carousel for example people who may have updated CoBlocks but have not yet edited an exiting post with an old post-carousel block. That old block would have old markup and should likely still load the old script. 

We will have a need to keep some jQuery around that is used in a deprecated manner.

Reply:
That being said. When I was looking at this block on the master branch earlier today the slick carousel is still working but its sure got some quirks like overlapping text for example.

Reply:
Thanks for testing this out. Resolving this comment.